back/engines/commercial/bulk_import_ideas/app/services/bulk_import_ideas/exporters/idea_pdf_form_exporter.rb

```diff
@@ -99,7 +99,8 @@ def render_tenant_logo(pdf)
       logo = @app_configuration.logo&.medium
       return if logo.blank? || logo.url&.include?('.gif')
 
-      pdf.image URI.open logo.to_s
+      logo_image = StringIO.new logo.file.read
+      pdf.image logo_image
```

Comment:
I've had to change the way files are read. Hopefully this also works when the files are remote and on AWS. Need to dig into this.

Reply:
If you want to test it locally, it seems you just need to set `USE_AWS_S3_IN_DEV=true` in `back-safe.env`.